marge/job.py

```diff
@@ -104,13 +101,13 @@ def add_trailers(self, merge_request):
 
         # add Tested-by
         should_add_tested = (
-            self._options.add_tested and
-            self._project.only_allow_merge_if_pipeline_succeeds and
-            self._options.fusion is Fusion.rebase
+            self._options.add_tested
+            and self._project.only_allow_merge_if_pipeline_succeeds
+            and self._options.fusion is Fusion.rebase
```

Comment:
Similar to https://github.com/hiboxsystems/marge-bot/pull/18#discussion_r1065636296, I think. I'll try to :see_no_evil: for now. :grin: